### DISCUSSION TITLE
5th attempt at getting btcpay to work

### DISCUSSION THREAD

**Discussion by ItchyPong** [btcpayserver/btcpayserver#2236] [Technical Support]
I have just tried again to try and install btcpay on woo commerce. First I tried the auto lunanode option, followed vid etc  and every time I wait and click my btc server link I get "site cannot be reached " or " timed out" I have tried different browsers and turning all anti virus off etc 

I gave up!   I then tried doing it from scratch via create vm create volume etc etc but then when i try and log into the console no matter what i type the password is rejected . 

I gave up! 

I tried to then use ssh key  . This fails also ! 

I have spent near 4 hours today and 5 yesterday just trying to get btcpay to run on woocommerce.  

Before I managed to get it to start and waited 2 days for the node to sort it self out. Then I changed my email and was locked out and to unlock it was like rocket science so I had to destroy vm etc and start over . Please can some one tell me what i am doing wrong ? Why is this so hard ?

**Comment by ItchyPong** [comment]
tried again and am now getting  

Your connection is not private
Attackers might be trying to steal your information from btcpay.vitalherbs.co.uk (for example, passwords, messages or credit cards). Learn more
NET::ERR_CERT_AUTHORITY_INVALID

**Reply by pavlenex** [reply]
Please don't open multiple discussions, it's really making it hard for us to follow what's going on.

Try `btcpay-down.sh` then `btcpay-up.sh` for some reason SSL certificate wasn't issued, you may want to check it's logs to see why. Best is to come to our chat.btcpayserver.org so someone can help you through the process.

You might as well wipe that LunaNode instance and start over if there's nothing of value stored there.

**Reply by ItchyPong** [reply]
Ok Ill do that thanks just getting very frustrated ! deleted all vms etc
